tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -538,7 +538,7 @@ public:
    /// \brief Save selected columns to disk, in a new TTree `treename` in file `filename`.
    /// \param[in] treename The name of the output TTree.
    /// \param[in] filename The name of the output TFile.
-   /// \param[in] columnNameRegexp The regular expression to match the column names to be selected. The presence of a '^' and a '$' at the end of the string is implicitly assumed if they are not specified. See the documentation of TRegexp for more details. An empty string signals the selection of all columns.
+   /// \param[in] columnNameRegexp The regular expression to match the column names to be selected. The presence of a '^' and a '$' at the end of the string is implicitly assumed if they are not specified. The dialect supported is PCRE via the TPRegexp class. An empty string signals the selection of all columns.
```

Comment:
In the documentation of `Snapshot` we give a couple of example usages: it could be useful to show how to select `muon_.*|electron_.*` -- mostly because people will confuse globbing syntax (`muon_*`) with regexp syntax (`muon_.*`)

Reply:
yes, even if we say PCRE, it would be nice to show an example.